js/index.js

```diff
@@ -1,7 +1,95 @@
+console.log("I'm ready!");
+
+// 1.1 Create a variable `hacker1` with the driver's name.
+
+// 1.2 Print `"The driver's name is XXXX"`.
+
+// 1.3 Create a variable `hacker2` with the navigator's name.
+
+// 1.4 Print `"The navigator's name is YYYY"`.
+
 // Iteration 1: Names and Input
+let hacker1 = "Matteo"
+let hacker2 = "Arnold"
+  console.log('The driver\'s name is ' + hacker1 + '.');
+  console.log('The driver\'s name is ' + hacker2 + '.');
+
```

Comment:
You can use backticks and template strings here. Like `The driver's name is ${hacker1}`